src/Sql/AbstractSql.php

```diff
@@ -304,33 +295,26 @@ protected function processSubSelect(
 
             // copy count
             $this->processInfo['subselectCount'] = $decorator->processInfo['subselectCount'];
+
             return $sql;
         }
 
         return $decorator->buildSqlString($platform, $driver, $parameterContainer);
     }
 
-    /**
-     * @param Join[] $joins
-     * @param PlatformInterface $platform
-     * @param null|DriverInterface $driver
-     * @param null|ParameterContainer $parameterContainer
-     * @return null|string[] Null if no joins present, array of JOIN statements
-     *     otherwise
-     * @throws Exception\InvalidArgumentException for invalid JOIN table names.
-     */
     protected function processJoin(
-        Join $joins,
-        PlatformInterface $platform,
-        DriverInterface $driver = null,
-        ParameterContainer $parameterContainer = null
-    ) {
+        Join               $joins,
+        PlatformInterface  $platform,
```

Comment:
again here - parameters are not aligned (I can see the same many timex below, not going to comment on the same)